src/command/dns-command.ts

```diff
@@ -133,7 +133,7 @@ export class DnsCommand implements CommandInterface<DnsOptions> {
 			const cmdResult = await cmd;
 
 			if (cmdResult.stdout.length === 0) {
-				logger.error('Successful stdout is empty', cmdResult);
+				logger.error('successful stdout is empty', cmdResult);
```

Comment:
I don't see why we're changing the casing here at all, especially when the sentences then end with full stops. Don't think that was the point of this PR.

Reply:
We are changing the case here for the consistent casing across the whole app. Idea is to have as clean messages as possible so it is:
```
[2023-06-12 16:50:50] [ERROR] [general] successful stdout is empty
```
For multiple sentences we are separating them with dot and a capital letter:
```
[2023-06-12 16:50:50] [ERROR] [general] detected an outdated probe. Restarting
```
 If some other format suits better I am open to suggestions.

Reply:
I just don't see how "detected an outdated probe. Restarting" is cleaner than "Detected an outdated probe. Restarting."

The problem was logging too much data, not this.

Reply:
Too much logging is fixed by updates in status-manager.ts and api-error-handler.ts in #154. And that is an update to make all messages look similar.

Reply:
That's all fine, but I don't understand why are you changing the casing instead of keeping it how it was, because this particular change seems to be for the worse.

Reply:
I just liked that format the most. If `[2023-06-12 16:50:50] [INFO] [general] Such format.` is better lets update to it instead.

Reply:
Ok, let's update.